lib/hammer_cli_katello/acs_bulk_actions.rb

```diff
@@ -0,0 +1,38 @@
+module HammerCLIKatello
+  class AcsBulkActionsCommand < HammerCLIKatello::Command
+    desc 'Modify alternate content sources in bulk'
+    resource :alternate_content_sources_bulk_actions
+
+    class RefreshAllCommand < HammerCLIKatello::SingleResourceCommand
+      action :refresh_all_alternate_content_sources
+      command_name 'refresh-all'
+      desc _("Refresh all alternate content sources")
+      success_message _("Successfully refreshed all alternate content sources")
+      failure_message _("Could not refresh all alternate content sources")
+
+      build_options
+    end
+
+    class RefreshCommand < HammerCLIKatello::SingleResourceCommand
+      action :refresh_alternate_content_sources
+      command_name 'refresh'
+      desc _("Refresh alternate content sources")
+      success_message _("Successfully refreshed specified alternate content sources")
+      failure_message _("Could not refresh the specified alternate content sources")
+
+      build_options
+    end
```

Comment:
These SingleResourceCommand should be available in ACS command directly, no? Do we need these here?

Reply:
In theory, yes. Since bulk ACS actions were a separate category in the API, I figured it would make sense to split this functionality off in a subcommand. If they were directly under the `alternate-content-source` command the refresh and destroy commands would need to be renamed something like `bulk-refresh` to not conflict with the existing ACS refresh and destroy commands.